hazelcast/src/main/resources/cluster.sh

```diff
@@ -71,50 +100,41 @@ fi
 
 command -v curl >/dev/null 2>&1 || { echo >&2 "Cluster state script requires curl but it's not installed. Aborting."; exit -1; }
 
+URL_BASE="${URL_SCHEME}://${ADDRESS}:${PORT}/hazelcast/rest/management/cluster"
+CURL_CMD="curl $CURL_ARGS"
+
 if [ "$OPERATION" != "get-state" ] && [ "$OPERATION" != "change-state" ] && [ "$OPERATION" != "shutdown" ] &&  [ "$OPERATION" != "force-start" ] && [ "$OPERATION" != "partial-start" ] && [ "$OPERATION" != "get-cluster-version" ] && [ "$OPERATION" != "change-cluster-version" ]; then
     echo "Not a valid cluster operation, valid operations  are 'get-state' || 'change-state' || 'shutdown' || 'force-start' || 'partial-start' || 'get-cluster-version' || 'change-cluster-version'"
-    exit 0
+    exit 2
 fi
 
-MSG_FORBIDDEN="Please make sure you provide valid group name (or valid username/password combination when the Hazelcast security is enabled)."
-
 if [ "$OPERATION" = "get-state" ]; then
-    echo "Getting cluster state on ip ${ADDRESS} on port ${PORT}"
-	request="http://${ADDRESS}:${PORT}/hazelcast/rest/management/cluster/state"
- 	response=$(curl --data "${GROUPNAME}&${PASSWORD}" --silent "${request}");
+    echo "Getting cluster state on address ${ADDRESS}:${PORT}"
+    response=$(${CURL_CMD} --data "${GROUPNAME}&${PASSWORD}" "${URL_BASE}/state");
```

Comment:
minor: is the trailing semicolon necessary here? Occurs in more places, and I see it's been in the older version, too.